shell/browser/ui/tray_icon_cocoa.mm

```diff
@@ -465,16 +265,15 @@ - (BOOL)shouldHighlight {
 
 TrayIconCocoa::~TrayIconCocoa() {
   [status_item_view_ removeItem];
-  if (menu_model_)
-    menu_model_->RemoveObserver(this);
 }
 
 void TrayIconCocoa::SetImage(const gfx::Image& image) {
   [status_item_view_ setImage:image.IsEmpty() ? nil : image.AsNSImage()];
 }
 
 void TrayIconCocoa::SetPressedImage(const gfx::Image& image) {
-  [status_item_view_ setAlternateImage:image.AsNSImage()];
+  [status_item_view_
+      setAlternateImage:image.IsEmpty() ? nil : image.AsNSImage()];
```

Comment:
just use `image.AsNSImage()`, the methods returns `nil` when `IsEmpty()`

Reply:
https://cs.chromium.org/chromium/src/ui/gfx/image/image.cc?sq=package:chromium&g=0&l=578-580